python/tvm/relay/frontend/pytorch.py

```diff
@@ -250,7 +250,9 @@ def _impl(inputs, input_types):
         channels = weight_shape[0]
         groups = int(inputs[8])
 
-        if groups > 1:
+        # weight_shape[1] is always in_channels // groups
+        # For depthwise, in_channels == groups
```

Comment:
This comment is a little confusing, Make it clear that we're checking if this is a depthwise conv and that if weight_shape[1] != 1, then its a group conv.

Reply:
Added more comments. Also explained why we need reshape for depth wise case.